### DISCUSSION TITLE
Dendron Tutorial Feedback

### DISCUSSION THREAD

**Discussion by kevinslin** [dendronhq/dendron#3266] [General]
We're constantly iterating to make the tutorial better. Please report here if anything about the Dendron tutorial does not work or is unclear!

**Comment by HuangJunye** [comment]
The getting started tutorial is great! I just want to mention a small typo in "Next Steps" section. 

- "Appy Template" => "Apply Template"

<img width="520" alt="image" src="https://user-images.githubusercontent.com/7631333/186885239-220fb948-7c42-47f0-b7cf-5641ac20661e.png">

**Reply by kevinslin** [reply]
thanks for this. we'll fix this in the next release!

**Comment by CoryOsmosisJones** [comment]
It wasn't immediately clear to me that the note preview was on the right side and I read mostly through the tutorial markup before realizing it was a lot easier to read the preview on the right. Perhaps make some mention to the two screens showing the same things in different representations.

My mistake was further deepened by misunderstanding the note "We will teach you at the end how to unlock the preview". I thought that meant the preview was locked as was not showing the result of the markup and that I would learn how to preview it at the end of the tutorial.

**Reply by kevinslin** [reply]
I can see how that would be confusing. we'll add a notice at the top about the preview!

**Comment by tlylt** [comment]
Hi there,

- navigating to link via CTRL + Enter doesn't seem to work (I'm using Windows 10 on VScode)
![image](https://user-images.githubusercontent.com/41845017/205425047-405927cd-c0ac-4df7-ae2b-725735660b80.png)
- inconsistency when referencing keyboard shortcuts is somewhat confusing: e.g. in `<CTRL>-<TAB>` vs `<CTRL>+<ENTER>` was confused for a second and thought that maybe there's some difference in `-` and `+`
![image](https://user-images.githubusercontent.com/41845017/205425117-05140599-2c22-42f7-abd1-52567bbf85a0.png)
![image](https://user-images.githubusercontent.com/41845017/205425168-d579954d-4892-4d4e-9c0e-6e509ea91155.png)

**Reply by fprotazio** [reply]
I was going to report exactly the same thing in the "Navigate a link" section but since you reported already, I am commenting here. 

The only different for me is that I am using macOS and command `<CMD>+<ENTER>` did not work too. Furthermore, the `<ENTER>` equivalent on MacBooks is `<RETURN>`

**Reply by jonathandavidlewis** [reply]
This instruction was not accurate in my case. I am on MacOS. Command + enter did not work. It was the CTRL key, not COMMAND.

on MacOS: <CTRL> + <RETURN> opens the link

**Reply by KAMDEMJERRY** [reply]
The command also did not work for me. I am using Ubuntu22.04 on vscode, but the CTRL+CLICK did work

**Reply by rachelavril** [reply]
same here ^^

**Comment by OneCricketeer** [comment]
RE

> `NOTE: when you rename a note, Dendron updates all links and references of the original note being renamed. Try switching back to [[tutorial]] to see the updated link!`

This did not happen for me. 

I did replace the whole name (`tutorial.one`) with `my-note` originally, so maybe my error, and that messed up the cache. But when I went back and renamed it back to `tutorial.one`, that fixed the link, but then renaming to `my-note.one` did not replace anything that I could see in `tutorial`. 

But also, I was deleting sections as I went, and wasn't actively saving the file (I have auto-save disabled), so maybe it cannot replace links for unsaved files?

**Reply by bmda** [reply]
The link did not update for me, either.

**Comment by JamesHoran** [comment]
"control + k v" is not doing anything. VS Code says it's not a valid command, while your documentation seems to say it is (https://wiki.dendron.so/notes/ldt8dqn41k80emerw5w88by/). If I'm doing this wrong please let me know, but pressing "k" followed by "v" all while holding down the control key is not working for me. I want to be able to toggle the preview using keyboard shortcuts, but it just isn't working now

**Comment by sensibubs** [comment]
I'm interested in importing my Obsidian notes, and noticed the link right down the bottom (Just above the Community heading) doesn't do anything
When I copied that line, the text comes out as "Coming from Obsidian? Click [here](https://github.com/dendronhq/dendron/discussions/3266) to import your Obsidian notes (or any markdown notes) into Dendron to see how they look.",
but clicking on "here" doesn't do anything for me

**Comment by alelom** [comment]
I'm using [Visual Studio Keymap](https://marketplace.visualstudio.com/items?itemName=ms-vscode.vs-keybindings) as I came to VSCode from its bigger brother Visual Studio, and I'm used to its keybinds.

With that extension installed, the `CTRL+L` shortcut suggested in the tutorial doesn't work.
Perhaps some instructions should be added for people with keybinds different than the original, at least to configure Dendron properly.
